src/octree/visualize.rs

```diff
@@ -112,11 +115,17 @@ impl Octree {
             gl::READ_ONLY,
             gl::R32UI,
         );
-        helpers::bind_3d_image_texture(
+        //helpers::bind_3d_image_texture(
+            //4,
+            //self.textures.brick_pool_normals,
+            //gl::READ_ONLY,
+            //gl::RGBA8,
+        //);
+        helpers::bind_image_texture(
             4,
```

Comment:
We should use index 5 for this since 4 is used for normals. They are used in the geometry shader